### PR TITLE
feat: also build 1.9.x and 1.8.x as part of nightly builds

### PR DESCRIPTION
**Pull request by lburgazzoli**
**Release Note**
```release-note
NONE
```

### CODE REVIEW

**Comment by lburgazzoli** [comment]
@squakez I know there is https://github.com/apache/camel-k/issues/3260 but in the meantime it would be nice to have some nightly build also for the 1.9.x and 1.8.x so people have the chance to test also such nightly release after backports before a release is cut.

let me know if it make sense, if not, flee free to close this pr

File: .github/workflows/release-workflow.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+          echo "IMAGE_NAME=$IMAGE_NAME" >> $GITHUB_ENV
+
+          MAVEN_REPOSITORY=$(make get-staging-repo)
+          echo "Using MAVEN_REPOSITORY=$MAVEN_REPOSITORY"
+          echo "MAVEN_REPOSITORY=$MAVEN_REPOSITORY" >> $GITHUB_ENV
+
+      - name: Login to Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
```
Comment (squakez) [inline .github/workflows/release-workflow.yml:80]: I guess it should login to docker hub instead

Comment (lburgazzoli) [reply .github/workflows/release-workflow.yml]: fixed

**Comment by lburgazzoli** [comment]
> Nice stuff. We probably will need some live test on the `release` but it's good for me.

yeah, I think that this is really an interim solution till we get  #3260

**Comment by squakez** [comment]
@lburgazzoli feel free to turn it into a ready PR if you're done and we can see the result in during next nightly build.

**Comment by squakez** [comment]
It seems the workflow failed: https://github.com/apache/camel-k/actions/runs/2836308573/workflow
```
error parsing called workflow "./.github/workflows/release-workflow.yaml": failed to fetch workflow: workflow was not found.
```
but the file is there. Not sure why it complains.

**Comment by lburgazzoli** [comment]
damn, it's been the n-th time I'm doing this mistake .... 
opened a pr https://github.com/apache/camel-k/pull/3535

**Pull request merged**